chandra_aca/tests/test_star_probs.py

```diff
@@ -58,7 +58,8 @@ def test_acq_success_prob_spoiler():
 
 def test_acq_success_prob_color():
     p_0p7color = .4294  # probability multiplier for a B-V = 0.700 star (REF?)
-    color = [0.6, 0.7, 1.5]
+    color = [0.6, 0.699997, 0.69999999, 0.7, 0.700001, 1.5]
```

Comment:
The test change here is largely just checking that np.isclose works, but it is reassuring.